benches/lib.rs

```diff
@@ -41,6 +41,19 @@ mod tests {
       }
     });
   }
+  #[bench]
+  fn bench02_pooled_scoped_allocation_speed(b: &mut Bencher) {
```

Comment:
I started numbering the benchmark names so that related test results would be shown side-by-side. (e.g. `bench01_standard_allocation_speed` and `bench02_pooled_allocation_speed`.) It works, but in situations like this where you'd like to add another test, it requires re-numbering all of the functions. Any suggestions for a better approach?